10/collection-meta.yaml

```diff
@@ -309,6 +309,12 @@ collections:
       - rndmh3ro
       - schurzi
     repository: https://github.com/T-Systems-MMS/ansible-collection-icinga-director
+    removal:
+      major_version: 11
+      reason: renamed
+      new_name: telekom_mms.icinga_director
+      announce_version: 10.5.0
```

Comment:
Why are announce versions different for each major version? Why isn't is also `9.0.0a1` here? Isn't that when it was first announced?

Reply:
Basically this says when the announcement appears in the changelog of this major Ansible version. We need this information to be able to render the changelog.

Reply:
Got it